build.gradle

```diff
@@ -2,9 +2,9 @@ apply plugin: 'groovy'
 apply plugin: 'eclipse'
 apply plugin: 'maven'
 
-sourceCompatibility = 1.5
-targetCompatibility = 1.5
-version = '0.8.4'
+sourceCompatibility = JavaVersion.VERSION_1_7
+targetCompatibility = JavaVersion.VERSION_1_7
```

Comment:
Do we have to set the compatibility level to version 1.7? If we have a choice, we may want to support as old as possible in compatibility.